docs/designers-developers/developers/backward-compatibility/README.md

```diff
@@ -1 +1,57 @@
 # Backward Compatibility
+
+Gutenberg follows the WordPress backward compatibility principles. This means backward compatibility for production public API is guaranteed. In some rare occasions breaking backward compatibility is unavoidable, in that case, the breakage should be as small as possible and be documented as clearly as possible to third-party developers using dev notes.
```

Comment:
> the WordPress backward compatibility principles

Is there a broader document about that somewhere?

Reply:
I looked quickly but didn't find any 🤔